store/sql_spinmint_store.go

```diff
@@ -15,19 +15,17 @@ type SQLSpinmintStore struct {
 }
 
 func NewSQLSpinmintStore(sqlStore *SQLStore) SpinmintStore {
-	s := &SQLSpinmintStore{sqlStore}
-
-	for _, db := range sqlStore.GetAllConns() {
-		table := db.AddTableWithName(model.Spinmint{}, "Spinmint").SetKeys(false, "InstanceId")
-		table.ColMap("InstanceId").SetMaxSize(128)
-	}
-
-	return s
+	return &SQLSpinmintStore{sqlStore}
 }
 
 func (s SQLSpinmintStore) Save(spinmint *model.Spinmint) (*model.Spinmint, error) {
-	if err := s.GetMaster().Insert(spinmint); err != nil {
-		if _, err := s.GetMaster().Update(spinmint); err != nil {
+	insertQuery := `INSERT INTO Spinmint (InstanceId, RepoOwner, RepoName, Number, CreatedAt)
+        VALUES (:InstanceId, :RepoOwner, :RepoName, :Number, :CreatedAt)`
+	if _, err := s.dbx.NamedExec(insertQuery, spinmint); err != nil {
```

Comment:
We can directly use the string inside the `NamedExec` call itself. Similar to how other queries are done.